packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -20,25 +20,47 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * Indicates if the DetailList should be a description list for label-value pairs.
```

Comment:
Do we need to include default value here, or does that already get populated in the props table?

Reply:
@kolkheang - I believe the default value is populated from the defaultProps variable.

![Screen Shot 2023-06-05 at 4 59 21 PM](https://github.com/cerner/terra-clinical/assets/55503680/188585c1-f718-4f88-8966-fd48abf58090)